server/commands.py

```diff
@@ -5667,13 +5535,11 @@ def ooc_cmd_unilock(client: ClientManager.Client, arg: str):
                                             (note the ,\ in the command).
     """
 
-    areas = arg.split(', ')
-    if len(areas) > 2 or arg == '':
-        raise ArgumentError('This command takes one or two arguments.')
-    if len(areas) == 2 and not client.is_staff():
+    Constants.assert_command(client, arg, parameters='&1-2', split_commas=True)
+    if len(arg) == 2 and not client.is_staff():
```

Comment:
This does not properly check that a non-staff player did not try to use two area names with /unilock